### PR TITLE
Support .netstandard2.1

### PR DESCRIPTION
**Pull request by gldeng**

### CODE REVIEW

File: src/AElf.Client/Services/HttpService.cs [hunk 1]
```diff
@@ -46,8 +47,14 @@ public HttpService(int timeoutSeconds)
             HttpStatusCode expectedStatusCode = HttpStatusCode.OK)
         {
             var response = await GetResponseAsync(url, version, expectedStatusCode);
-            var stream = await response.Content.ReadAsStreamAsync();
-            return await JsonSerializer.DeserializeAsync<T>(stream);
+            await using var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false);
```
Comment (argfoo-web3) [inline src/AElf.Client/Services/HttpService.cs]: Remove ConfigureAwait(false)

File: src/AElf.Client.Protobuf/AElf.Client.Protobuf.csproj [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
-        <ImplicitUsings>enable</ImplicitUsings>
+        <TargetFrameworks>netstandard2.1;net6.0</TargetFrameworks>
         <Nullable>enable</Nullable>
         <PackageId>AElf.Client.Protobuf</PackageId>
         <PackageDescription>Protobuf using by AElf C# sdk.</PackageDescription>
         <Company>AElf</Company>
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <Version>1.3.1</Version>
```
Comment (argfoo-web3) [inline src/AElf.Client.Protobuf/AElf.Client.Protobuf.csproj]: Remove Version tag

File: src/AElf.Client.Protobuf/AElf.Client.Protobuf.csproj [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="AElf.Types" Version="1.0.0" />
+        <PackageReference Include="AElf.Types" Version="1.5.0" />
```
Comment (argfoo-web3) [inline src/AElf.Client.Protobuf/AElf.Client.Protobuf.csproj]: Change Version for AElf.Types to 1.3.0

**Comment by argfoo-web3** [comment]
Build will fail due to 1.3.0 for AElf.Cryptography and AElf.Types not supporting .Net Standard 2.1. We will need to make sure we create a new nuget version for this PR (https://github.com/AElfProject/AElf/pull/3433) first before we merge this.

**Review by jason-aelf — APPROVED** [review]

**Pull request merged**